enrollment-server-onboarding/src/main/resources/application.properties

```diff
@@ -72,19 +72,20 @@ powerauth.service.security.clientSecret=
 # Onboarding Process Configuration
 enrollment-server-onboarding.onboarding-process.enabled=false
 enrollment-server-onboarding.onboarding-process.otp.length=8
-enrollment-server-onboarding.onboarding-process.otp.expiration=PT30S
+enrollment-server-onboarding.onboarding-process.otp.expiration=30s
 enrollment-server-onboarding.onboarding-process.otp.max-failed-attempts=5
-enrollment-server-onboarding.onboarding-process.otp.resend-period=PT30S
-enrollment-server-onboarding.onboarding-process.activation.expiration.seconds=300
-enrollment-server-onboarding.onboarding-process.verification.expiration.seconds=86400
+enrollment-server-onboarding.onboarding-process.otp.resend-period=30s
+enrollment-server-onboarding.onboarding-process.expiration=3h
+enrollment-server-onboarding.onboarding-process.activation.expiration=5m
+enrollment-server-onboarding.onboarding-process.verification.expiration=1h
 enrollment-server-onboarding.onboarding-process.max-processes-per-day=5
 
 # Identity Verification Configuration
 enrollment-server-onboarding.identity-verification.enabled=false
-enrollment-server-onboarding.identity-verification.data-retention.hours=1
-enrollment-server-onboarding.identity-verification.expiration.seconds=300
+enrollment-server-onboarding.identity-verification.data-retention=1h
```

Comment:
Here, it is correctly without _hours_ This works only due to default value https://github.com/wultra/enrollment-server/pull/266/files#r923018751 I will fix it in a separate PR.

Reply:
https://github.com/wultra/enrollment-server/pull/279